scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -1279,8 +1279,7 @@ def __create_log_config(
         @param path: the path of the log file being configured
         @param attributes: Any attributes to include as part of the log_config['attributes']
         @param base_config: A base set of configuration options to build the log_config from
-        @type parser: str
-        @type path: str
+        @type path: six.text_type
```

Comment:
Add back in `@type` for `default_parser`

Reply:
Oops, the `parser` was actually `default_parser`, so I had to rename it, instead of removing. Thanks.